lib/reporting/reports/suppliers/helpers/line_items_access_helper.rb

```diff
@@ -27,24 +27,22 @@ def item_order_cycle(line_item)
 
           def suppliers_adjustments(line_item, adjustment_type = 'EnterpriseFee')
```

Comment:
I am wondering if we should rename this `supplier_adjustments_by_type` and remove the default `adjustment_type` I think it would make the code easier to understand. Anyway that shouldn't block merging.